### ISSUE TITLE
Extra 'n' on Component Usage section

### ISSUE THREAD

**Issue by uNkNowN92-git** [SupremeTechnopriest/react-idle-timer#125]
Please remove extra 'n' on the Component Usage section.

this.handleOnAction = this.handleOn~~n~~Action.bind(this)
this.handleOnActive = this.handleOn~~n~~Active.bind(this)
this.handleOnIdle = this.handleOn~~n~~Idle.bind(this)

**Comment by alexpapworth** [comment]
Thought this was a deliberate choice!

If anyone else gets a `TypeError: Cannot read property 'bind' of undefined` error when copying code from the README, this is why.

```
   5 | constructor(props) {
   6 |   super(props)
   7 |   this.idleTimer = null
>  8 |   this.handleOnAction = this.handleOnnAction.bind(this)
   9 |   this.handleOnActive = this.handleOnnActive.bind(this)
  10 |   this.handleOnIdle = this.handleOnnIdle.bind(this)
  11 | }
```

**Comment by SupremeTechnopriest** [comment]
Oof my bad!  Let me get that fixed. :)

**Comment by SupremeTechnopriest** [comment]
@uNkNowN92-git, @alexpapworth, Good to go!  Published v`4.3.6` so the readme on NPM gets updated too.

**Comment by alexpapworth** [comment]
Awesome, thanks!
